cmake/common.cmake

```diff
@@ -85,6 +85,10 @@ if(DEBUG_CONFIG)
   add_definitions(-DDEBUG_CONFIG)
 endif()
 
+option(METRICS "Enable gathering of transaction rate metrics" OFF)
+if(METRICS)
```

Comment:
I'm not convinced that the tiny cost of metrics tracking warrants a compile-time flag. I think we ought to default to metrics enabled, and possibly make it a runtime flag to disable them.